components/help.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from "react";
+
+import { LanguageContext } from "@contexts/translator";
+
+// eslint-disable-next-line react/prop-types
+const Help = ({ close }) => {
+  const { getTranslation } = useContext(LanguageContext);
+
+  return (
+    <div id="help-info" className="help-info padded spaced-lines">
+      <button
```

Comment:
I want to change this to using the icon or in general having consistency across the close buttons throughout, but doesn't need to happen now and I suppose turning Info into an actual page will make it a moot point

Reply:
Good to know! Yeah `X` felt like a weird button copy decision but agreed, lets revisit this in a later PR!